apps/browser-extension-wallet/src/hooks/useNftSearch.ts

```diff
@@ -0,0 +1,43 @@
+import { useState } from 'react';
+import { NftItemProps } from '@lace/core';
+import { AssetOrHandleInfoMap } from './useAssetInfo';
+import { Cardano } from '@cardano-sdk/core';
+import debounce from 'lodash/debounce';
+
+const DEBOUNCE_TIME = 1000;
+
+interface NftSearchResultProps {
+  isSearching: boolean;
+  filteredResults: NftItemProps[];
+  handleSearch: (items: NftItemProps[], searchValue: string) => void;
+}
+
+export const searchNfts = (
+  data: NftItemProps[],
+  searchValue: string,
+  assetsInfo: AssetOrHandleInfoMap
+): NftItemProps[] =>
+  data.filter(
+    (item) =>
+      item.name.toLowerCase() === searchValue.toLowerCase() ||
```

Comment:
IMO it would have better experience if we searched for a partial string
```suggestion
      item.name.toLowerCase().includes(searchValue.toLowerCase()) ||
```

Reply:
How did we leave it on assets?

Reply:
Good question. IMO we should use `includes` in both places.